blog/_posts/2017-09-23-RGSoC- the grand finale.md

```diff
@@ -0,0 +1,99 @@
+---
+title: RGSoC- the Grand Finale.
+layout: post
+created_at: Fri September 23 2017
```

Comment:
Hi, thanks for the PR. Can you update the date at all 3 places with today's date : Tue October 3, 2017? Also, please make sure that permalink and filename is same.